src/lib/store/remote/api.js

```diff
@@ -140,10 +140,20 @@ exports.getSinceNr = function(state) {
 // changes since the beginning, but this behavior might be adjusted,
 // e.g for a filtered bootstrap.
 //
+// As several firewalls and anti virus software strip out Accept-Encoding
+// request headers, ~15% of users end up with much bigger download sizes
+// than needed. So we created a workaround to force gzip via url param,
+// and fallback to normal request if it fails:
+// https://github.com/hoodiehq/hoodie.js/issues/423
+//
 exports.bootstrap = function(state) {
   state.isBootstrapping = true;
   state.remote.trigger('bootstrap:start');
-  return state.remote.pull()
+  return state.remote.pull({forceGzip: true})
```

Comment:
> And I don't see a scenario where we would need something like gzip: false

See my note below about `gzip: off` and `gzip: headers`.

Debugging comes mind, too.

If this isn’t convincing: I don’t like camel case in url params. Can we do`force_gzip:true` or something similar?